app/controllers/publishers_controller.rb

```diff
@@ -72,12 +72,11 @@ def create
     verified_publisher = Publisher.by_email_case_insensitive(email).first
     if verified_publisher
```

Comment:
```ruby
 @publisher = Publisher.find_or_create_by(pending_email: email, email: nil)
```
In this action we first look to see if there is a valid publisher with the correct `pending_email` that also has `email: nil`.  If there is a such a publisher we assign `@publisher` to it.  If not, we assign `@publisher` to a new publisher.

Then we check if there is a valid publisher with that `email`.  If that's the case we overwrite `@publisher` to be the publisher with the valid `email`.

Shouldn't we be checking if there exists a valid publisher with the `email` first, and only run `Publisher.find_or_create_by(pending_email: email, email: nil)` if that's not the case?